### DISCUSSION TITLE
Open Minimap on startup

### DISCUSSION THREAD

**Discussion by Fireslayerjk** [blizzhackers/kolbot-SoloPlay#74] [Ideas]
To have the bot open the minimap before starting I made the following change. I like watching them with the map open.

In the file SoloPlay.js add the line "sendKey(sdk.keys.code.Tab);" right after "myPrint("starting run");".

**Comment by theBGuy** [comment]
So simpler than that is you just need to add, `Config.AutoMap = true` to
https://github.com/blizzhackers/kolbot-SoloPlay/blob/4505c40c5426bdcb6ab808d39117f60450709f6e/libs/SoloPlay/Functions/Globals.js#L499

**Comment by Fireslayerjk** [comment]
oh, good to know! I gave it a try and works great. Thanks for the tip!
